docs/proposals/1374-mc-inference-gateways/README.md

```diff
@@ -0,0 +1,165 @@
+
+### Failure Mode
+
+If the Endpoint Picker is unavailable and the failure mode is configured as “FailOpen”, we could take one of several approaches:
+
+#### Honor FailOpen configuration
+
+This seems to require the Gateway to be aware of at least some model server endpoints, which requires more endpoint propagation.
+
+#### Fail over to other cluster/Endpoint Picker
+
+In a world where there are multiple clusters/Endpoint Pickers to choose from, it may be desirable to fail over to another cluster. Ultimately if all Endpoint Pickers are unavailable, we may end up back at the same problem though of needing to be aware of model server endpoints.
+
+#### Consider FailOpen “Extended” support for multi-cluster
+
+Given the potential complexity of supporting a FailOpen mode for multi-cluster, we could consider this “Extended” or optional support.
+
+### Cluster/Endpoint Picker Selection
+
+It’s likely that each Gateway implementation will have some different logic here, but there will likely be at least two common paths here:
+
+#### Metrics from model server endpoints
+
+In the case where a Gateway is aware of all model server endpoints, it could theoretically also track metrics for each of these endpoints.
+
+#### Metrics from Endpoint Picker
+
+Since Gateways are ultimately deciding which Endpoint Picker to send traffic to, it could make sense for Endpoint Pickers to report back load/utilization data to the Gateway to help inform that decision. (This would reflect the utilization of model server Pods within the local InferencePool managed by each EPP).
+
+#### PreferClose/PreferLocal
+
+Local cluster by default, failover if out of capacity.
+
+### Theoretical Future Enhancement: Multi-Cluster Endpoint Pickers
+
+In the future,  a more advanced implementation could allow Endpoint Pickers to pick from endpoints in other clusters (relying on the same underlying infrastructure that propagates endpoints for this multi-cluster model). We're intentionally avoiding that from the initial scope as it's both more complicated to implement, and unlikely to be scalable given the need for Endpoint Pickers to have a very tight feedback loop (usually via frequent scraping of metrics) with each model server Pod in the InferencePool. Extending that model across clusters could become quite costly.
+
+**Pros**:
+
+- Reuses existing MCS model
+- Simplest possible API model
+- “Export” configuration lives on InferencePool and clearly applies to the entire pool, not just EPP
+- Can clearly reference an InferencePool in other clusters without having one locally
+
+**Cons**:
+
+- Does not reuse MCS API (unclear if this is a con)
+
+## Alternative 1: MCS API for EPP
+
+If we lean into the idea that the only thing a Gateway needs to know is the Endpoint Picker endpoints and what cluster(s) they're associated with, we could build this on top of the MCS API. With this approach, the Endpoint Picker is exposed with a Multi-Cluster Service:
```

Comment:
InferencePool is meant to be a replacement for a Service so it may seem counterintuitive for a user to create a Service to achieve multi-cluster inference.